packages/docs/content/components/button.mdx

```diff
@@ -242,13 +252,82 @@ A custom heading value can be sent for an analytics event by using the prop `ana
+- Ghost buttons are typically used when paired with a Solid or Outline button or to indicate lower priority actions. 
+
+</ThemeContent>
+
 ### Accessibility
 
+<ThemeContent onlyThemes={['healthcare']}>
+
+Consider when to use links vs. when to use buttons. Links and buttons are read differently using a screen readers. For screen readers buttons can be triggered by pressing the space bar or the enter key while links are only triggered by the enter key. This makes a difference to a screen reader user's flow. 
+
+</ThemeContent>
+
+
 - Buttons should display a visible focus state when users tab to them.
 - Create a button with a `button` or `a` element to retain the native click functionality. Avoid using `<div>` or `<img>` tags to create buttons. Screen readers don't automatically know either is a usable button.
 - When styling links to look like buttons, remember that screen readers handle links slightly differently than they do buttons. Pressing the `Space` key triggers a button, but pressing the `Enter` key triggers a link.
 - Dimmed or unavailable buttons should have the `disabled` attribute applied. This removes native click and keypress events from the button. It also prevents automated scanners from logging a low contrast error. Finally, it announces the button as "dimmed" or "disabled" to screen readers, offering users additional context.
 
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+### Content guidance
+
+- Describe what will happen, not the current state. 
+- Buttons always start with an action word describing the main thing users will do once selected. 
+  - Pair it with a noun to help it be clearer. Example: Get tips.
+  - If in a form, it can be just the action word if it's navigating between steps. Example: Cancel, Back, Next.
+- Use the least amount of words possible (no more than 6 with 2-4 being ideal).
+- Use sentence case for buttons. This conveys a friendlier tone. Note: Some tools and the homepage are using ALL CAPS. There are also a few examples of Title Case. We should change to sentence case for consistency across the site. 
+
+#### Writing content
+- Remove articles and use '&' instead of 'and.'
+- Do not capitalize articles. 
+- Avoid all caps unless the word is an acronym, like ZIP.
+- Avoid pronouns unless the future state provides the user with customized information. Example: "Get my results"
+- Limit branded terms, like tool names. As a general rule, we don't use tool names in web content or flow. Rather, content should outline what the user will get/can do when they click the button.
+
+#### Use this/not that
+
+##### Online forms & steps
+
+- Use: "Start" - to start a process or sequential steps. But generally better to use specific verbs to confirm what the user is starting. Example: "Create Account."
+  - Don't use: "Begin," "Go," "Enter," or "Submit."
+- Use: "Continue" - when moving through a process or sequential steps.
+  - Don't use: "Next."
```

Comment:
It's mentioned on line 290 that "Next" is acceptable (as are other single word button commands for forms)